packages/ripple-keypairs/src/index.ts

```diff
@@ -38,7 +38,7 @@ function generateSeed(
   const entropy = options.entropy
     ? options.entropy.slice(0, 16)
     : randomBytes(16)
-  const type = options.algorithm === 'ed25519' ? 'ed25519' : 'secp256k1'
+  const type = options.algorithm === 'ecdsa-secp256k1' ? 'secp256k1' : 'ed25519'
```

Comment:
Should this be tested against the enum?

Reply:
I've lost some of the context here. Which enum are you referring to?
In line 24, I'm using `secp256k1` imported from `'./signing-schemes/secp256k1'`

Are you referring to that import?